spec/src/main/asciidoc/preface.asciidoc

```diff
@@ -11,6 +17,10 @@ include::license-{license}.asciidoc[]
 
 Jakarta Contexts and Dependency Injection 3.0 (link:https://jakarta.ee/specifications/cdi/3.0/[CDI 3.0]) is an update to Jakarta Contexts and Dependency Injection 2.0 (link:https://jakarta.ee/specifications/cdi/3.0/[CDI 2.0]).
 
+Contexts and Dependency Injection 4.0 introduce a new feature with the CDI Lite specification.
+
+CDI Lite is a subset of CDI that doesn't require a CDI container at runtime and thus is a good candidate to develop cloud native or resources constraint applications.
```

Comment:
It does require CDI container at runtime :-) What it doesn't require to be at runtime is certain parts of CDI initialization, such as bean discovery and extension processing. Those can be moved to application build.

Reply:
Agreed, plus I think this part should probably be under `Major changes` section.
In fact, this `Foreword` section doesn't bring much valuable information - why don't we just merge it into `Major Changes` altogether?